### ISSUE TITLE
 Configuration for rule "comma-dangle" is invalid

### ISSUE THREAD

**Issue by reggi** [feross/standard#895]
Just added `npm run standard` to Travis CI and it's complaining about `comma-dangle`. 

Not sure if this is just me not installing deps correctly or apart of a larger issue.

Would apreciate any help.

```
> standard
standard: Unexpected linter output:
Error: standard:
    Configuration for rule "comma-dangle" is invalid:
    Value "[object Object]" must be an enum value.
Referenced from: /home/travis/build/project/node_modules/standard/eslintrc.json
    at validateRuleOptions (/home/travis/build/project/node_modules/eslint/lib/config/config-validator.js:116:15)
    at /home/travis/build/project/node_modules/eslint/lib/config/config-validator.js:163:13
    at Array.forEach (native)
    at Object.validate (/home/travis/build/project/node_modules/eslint/lib/config/config-validator.js:162:35)
    at load (/home/travis/build/project/node_modules/eslint/lib/config/config-file.js:518:19)
    at /home/travis/build/project/node_modules/eslint/lib/config/config-file.js:386:36
    at Array.reduceRight (native)
    at applyExtends (/home/travis/build/project/node_modules/eslint/lib/config/config-file.js:363:28)
    at Object.load (/home/travis/build/project/node_modules/eslint/lib/config/config-file.js:525:22)
    at loadConfig (/home/travis/build/project/node_modules/eslint/lib/config.js:64:33)
If you think this is a bug in `standard`, open an issue: https://github.com/feross/standard/issues
```

https://stackoverflow.com/questions/44164419/standard-js-in-travis-ci-configuration-for-rule-comma-dangle-is-invalid

**Comment by Flet** [comment]
Taking a look at the [eslint changelog](https://github.com/eslint/eslint/blob/master/CHANGELOG.md) I see that there was some `comma-dangle` bugs that were fixed:

>v3.8.1 - October 17, 2016
>
>681c78a Fix: comma-dangle was confused by type annotations (fixes #7370) (#7372) (Toru Nagashima)

Perhaps an older version of `eslint` was used that had this bug? Is it possible to clear the cache on Travis and try it again?

**Comment by Flet** [comment]
I'm going to close this for now but feel free to respond if the issue is still unresolved.
